Functions/Assertions/Exist.ps1

```diff
@@ -1,6 +1,7 @@
 
 function PesterExist($value) {
-    return (Test-Path $value)
+    $resolvedPath = $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($value)
```

Comment:
Yup, I realized that as well after the test failed on CI. How about just documenting it, adding more tests and keeping the assertion as it? https://github.com/pester/Pester/pull/348#issuecomment-101235064